users/enum.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum, unique
+
+@unique
+class User_type(Enum):
+    E = 'Employee'
+    M = 'Manager'
+    A = 'Admin'
+
+
+USER_TYPE_CHOICES = (
+    (User_type.E.value,    'Employee'),
+    (User_type.M.value,    'Manager'),
+    (User_type.A.value,    'Admin'),
+)
+assert set(s.value for s in User_type) == set(dict(USER_TYPE_CHOICES))
+
+@unique
+class Country(Enum):
```

Comment:
You could do it better: https://github.com/SmileyChris/django-countries
Just import this library and use `CountryField` and that's all.
It's better, because as a developers we don't want to worry about such things like countries when someone created stable, tested library for that. We don't want to reinvent the wheel and be up to date with current political situation. And this solution gives us wholesome solution for all countries, not just for chosen ones.

Reply:
Done